loadflow/loadflow-api/src/test/java/com/powsybl/loadflow/LoadFlowParametersTest.java

```diff
@@ -101,6 +113,9 @@ public void checkConfig() {
         boolean distributedSlack = true;
         LoadFlowParameters.BalanceType balanceType = LoadFlowParameters.BalanceType.PROPORTIONAL_TO_LOAD;
         LoadFlowParameters.VoltageInitMode voltageInitMode = LoadFlowParameters.VoltageInitMode.UNIFORM_VALUES;
+        boolean dcUseTransformerRatio = true;
+        Set<Country> countriesToBalance = new HashSet<>();
```

Comment:
`EnumSet.noneOf(Country.class)`? It is what we have in our legacy code.

Reply:
But indeed an empty list means nothing, the default behaviour is: all countries are involved in the slack distribution.

Reply:
I can include the whole country enum by default. Or just consider countriesToBalance as a filter which is disabled when empty. What is hadès behavior ?

Reply:
Empty means all